service/frontend/workflowHandler.go

```diff
@@ -4387,6 +4393,13 @@ func checkPermission(
 	return nil
 }
 
+func checkFailOverPermission(config *Config, domainName string) error {
+	if config.Lockdown(domainName) {
```

Comment:
Should we allow super users to perform failover when there is a lockdown?

Reply:
We don't plan on making this auth based. It will purely be a flipr config

Reply:
We talked about this yesterday. The only plan right now is to have this as a flipr config not any sort of auth based solution